content/_data/upgrades/2-346-1.adoc

```diff
@@ -0,0 +1,29 @@
+==== PNG and GIF icons replaced with SVG icons
+
+Scalable vector graphics images are used in Jenkins instead of the GIF and PNG images that were used previously.
+SVG images provide a better user experience and improve the Jenkins user experience.
+
+The GIF and PNG images have been removed from Jenkins core.
+
+Jenkins has removed a large number of GIF and PNG iamges.
+Ensure all your plugins are the most recent release before updating core, otherwise images may be missing.
+Check for new plugin versions immediately after the core update and install available updates.
```

Comment:
Optional non-blocking feedback for your consideration: This is a breaking change similar in scope to the Guava upgrade or tables-to-divs. In the upgrade guide for 2.332.1, we linked to a blog post about the Guava upgrade. That blog post contained a link to the open Jira issues associated with that migration as well as a list of instructions for how to report an issue relating to that migration. In my opinion it is worth considering a similar approach here, for the following reason. Users might be reassured that it is safe to upgrade if they can confirm that all plugins they use are either unaffected or in the list of affected plugins with a released fix; similarly, users might be justifiably concerned if they can confirm that a plugin they use is in the list of affected plugins without a released fix. Without this information, users might make a decision with incomplete information. One could link to [JENKINS-68251](https://issues.jenkins.io/browse/JENKINS-68251) for general tracking purposes. I do not think any special instructions are needed when filing issues, but we may wish to define a label associated with this migration (as we did for tables-to-divs and Guava) and encourage users to use that label for ease of tracking. This review feedback is non-blocking.